python/src/equistore/labels.py

```diff
@@ -251,6 +251,41 @@ def asarray(self):
     def __contains__(self, label):
         return self.position(label) is not None
 
+    @staticmethod
+    def arange(*args, name="name"):
+        """
+        Creates a very simple `Labels` object.
+
+        The resulting `Labels` only contain one label, whose name
+        is `name`, unless otherwise specified. The values of the
+        labels depend on a set of integer arguments, whose
+        meaning is the same as those of the `numpy.arange` function.
+        As a result, this function only accepts one to three such
+        integer parameters.
+        """
+
+        args_len = len(args)
+        if args_len == 0:
+            raise ValueError(
+                "please provide at least one integer for `Labels.arange()`"
+            )
+        if args_len > 3:
+            raise ValueError(
+                f"the maximum number of integer arguments accepted by \
+                `Labels.arange()` is 3. {args_len} were provided"
+            )
+        for arg in args:
+            if not np.issubdtype(type(arg), np.integer):
+                raise ValueError(
+                    "all numbers provided to `Labels.arange()` must be integers"
+                )
+
+        labels = Labels(
+            names=[name], values=np.arange(*args, dtype=np.int32).reshape(-1, 1)
+        )
```

Comment:
Also, do we want to allow for creation of Labels of > 1 dimension?

Perhaps something like this, passing a list of names and using the length of this to reshape the np.arange array

```suggestion

        if isinstance(names, str):
            names = [names]
        if not np.all([isinstance(n, str) for n in names]):
            raise TypeError("all names provided to `Labels.arange()` must be strings")

        labels = Labels(
            names=names, values=np.arange(*args, dtype=np.int32).reshape(-1, len(names))
        )
```

Reply:
Decided to keep to one dimension